person.rb

```diff
@@ -21,3 +28,35 @@ def of_age?
     false
   end
 end
+
+class PersonDecorator < Nameable
```

Comment:
- [ ] You need to add the `attr_accessor` method in order to be able to  access and set instance variable `nameable` from outside the class. 
```ruby
class Decorator < Nameable
    attr_accessor :nameable
```

Reply:
Thank you for the review @t-yanick, I have added the attr_accessor as recommended. Thank you